.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: dart
 dart_task:
+- test: --platform vm
+- test: --platform chrome
 - dartanalyzer
 - dartfmt
```

Comment:
Consider adding

```yaml
dart
- dev
- 2.4.0
```

Good to test on oldest supported SDK

Reply:
Done